CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - removed `init`, `loader` and `plugin` commands in favor [`create-webpack-app`](https://github.com/webpack/webpack-cli/tree/master/packages/create-webpack-app)
 - dropped support for `webpack-dev-server@v4`
 - minimum supported webpack version is `5.82.0`
+- removed `--define-process-env-node-env` in favor of `--config-node-env` option ([#4318](https://github.com/webpack/webpack-cli/pull/4318))
```

Comment:
changelog is auto generated so dont touch that

Reply:
Changes reverted in both `CHANGELOG.md` and `packages/webpack-cli/CHANGELOG.md`